articles/event-grid/resize-images-on-storage-blob-upload-event.md

```diff
@@ -125,8 +125,8 @@ An event subscription indicates which provider-generated events you want sent to
     | ------------ |  ------- | -------------------------------------------------- |
     | **Name** | imageresizersub | Name that identifies your new event subscription. | 
     | **Topic type** |  Storage accounts | Choose the Storage account event provider. | 
-    | **Subscription** | Your subscription | By default, your current subscription should be selected.   |
-    | **Resource group** | myResourceGroup | Select **Use existing** and choose the resource group you have been using in this topic.  |
+    | **Subscription** | Your Azure subscription | By default, your current Azure subscription should be selected.   |
```

Comment:
Was not sure if "subscription" meant Event Grid subscription or Azure subscription.